Automattic/wp-calypso
# Add two new solutions to the cancellation flow

#### Proposed Changes

* This PR adds two new solutions - Live Chat and Built By - to the cancellation flow and also applies fresh visual styles to the flow as a part of the Cancelation Reason/Solutions project. For more details, see pebzTe-kL-p2

#### Testing Instructions



* Try to cancel/remove your paid plan.
* You would be taken to the solutions based on the reason you picked. Visit pebzTe-nt-p2 to see the reasons and the relevant solutions. For example, you will see the `Downgrade to Monthly Plan` solution when you choose `Price/Budget > It's too expensive`.
  <img width="821" alt="화면_기록_2022-10-28_오후_9_01_03_mov" src="https://user-images.githubusercontent.com/212034/198583465-ed4c65c0-3a7c-4c8c-a351-030c3fbeafe2.png">
  <img width="800" alt="화면_기록_2022-10-28_오후_9_01_03_mov" src="https://user-images.githubusercontent.com/212034/198583562-94c2214f-577a-44c2-8f5d-daa426a8a516.png">
* Make sure the buttons work properly.
* For testing LiveChat, see pebzTe-sj-p2
  
#### Pre-merge Checklist






Related to #
